CMakePresets.json

```diff
@@ -45,6 +52,13 @@
         "CMAKE_CUDA_ARCHITECTURES": "87"
       }
     },
+    {
+      "name": "JetPack 7",
```

Comment:
JetPack 7 is not a known version according to the NVIDIA developer portal. The latest JetPack release at time of writing is [6.2](https://developer.nvidia.com/embedded/jetpack-sdk-62)

https://developer.nvidia.com/embedded/jetpack-archive

Reply:
It’s under testing. Well, we will do a new PR in the future. It’s coming with jetson thor that is supported in cuda 12.8